src/open_company_web/components/ui/add_topic_popover.cljs

```diff
@@ -46,12 +46,12 @@
 (defn on-click-in [owner options e]
   (.stopPropagation e))
 
-(defn get-state [{:keys [active-topics-list all-topics archived-topics] :as data} old-state]
+(defn get-state [{:keys [topic-order active-topics-list all-topics archived-topics] :as data} old-state]
   (let [topics-list (map name (keys all-topics))
```

Comment:
@belucid i guess this line is not needed anymore since topics-list was only used in place of topic-order